src/non_zero.rs

```diff
@@ -111,6 +111,18 @@ impl NonZero<Limb> {
 }
 
 impl<const LIMBS: usize> NonZero<Uint<LIMBS>> {
+    /// Creates a new non-zero integer in a const context.
+    /// # Panics
+    ///
+    /// Panics if the value is zero.
+    pub const fn const_new(n: Uint<LIMBS>) -> Self {
```

Comment:
In `const-oid` we call a similar function `new_unwrap`: https://docs.rs/const-oid/latest/const_oid/struct.ObjectIdentifier.html#method.new_unwrap

Reply:
Renamed in [7f6bbb1](https://github.com/RustCrypto/crypto-bigint/pull/602/commits/7f6bbb12fcef27464186b57cd08fd19358af36c8)